packages/cli/commands/upload.js

```diff
@@ -63,9 +63,15 @@ exports.handler = async options => {
   const uploadPromptAnswers = await uploadPrompt(options);
 
   const src = options.src || uploadPromptAnswers.src;
-  const dest = options.dest || uploadPromptAnswers.dest;
+  let dest = options.dest || uploadPromptAnswers.dest;
```

Comment:
Maybe missing it but does `dest` need to be reassigned?

Reply:
Nope. My bad, not sure how that got changed.

Reply:
Forgot to respond to this before fixing but yeah dest should be getting reassigned - this is to correct for the case when a user uploading a fields.js types the dest file as fields.js instead of .json.